components/AboutInfo.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { withRouter } from "next/router";
+
+const AboutInfo = ({ router }) => {
+  return (
+    <div className="container m-auto"> 
+       <div className="md:w-4/5 w-11/12  bg-gray-900 rounded-lg px-4 flex-wrap py-2 my-10 m-auto">
+            <h1 className="text-5xl text-yellow-500 text-center">About Us</h1>
+           <div className="lg:flex justify-between items-center ">
+                
+                <p className="text-xl   m-8 text-center">
+                GDSC CMRIT is a university-based community group for students interested in Google developer technologies. GDSC can take many forms-from just a few people getting together to watch some videos, to large gatherings with demos, tech talks, hackathons and conferences. We work on various technical domains and try to make use of Google Technologies to empower the audience around us.. By joining a GDSC, students grow their knowledge in a peer-to-peer learning environment and build solutions for local businesses and their communities.
+                </p>
+                <img src="/about_us.jpg"  className="md:w-80 sm: mx-auto block"></img>
+           </div>
+          
+        </div>     
+        {/* About US done */}
+        {/* Mission starts */}
+        <div className="lg:w-4/5 w-11/12  bg-gray-900 rounded-lg px-4  py-2  my-10 m-auto text-center">
+            <h1 className="text-5xl text-yellow-500">Our Mission</h1>
+            <div className="lg:flex justify-between items-start">
+                <div className="md:w-96 md:w-1/2 lg:mr-2 my-5 m-auto  bg-purple-600 bg-opacity-25 rounded-lg">
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_edu.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75  rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-green-600 text-black text-opacity-100 uppercase">Education</h3>
+                        <p className="mx-5 pb-5">
+                        GDSC CMRIT provides a platform for the growth and advancement of the student community in the field of technology by providing peer-to-peer learning environment by conducting , webinars and events , providing hands on experiences on topics like Web Development, Mobile App Development, Data Science, Blockchain,  Machine Learning, Contributing to Open Source and much more!</p>
+                    </div>
```

Comment:
This is very similar piece of code just differing in text, so you can create a separate component and just pass title, image and text as prop.
we use components to reduce code duplication.